LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/OTAAHelper/OTAAKeysGenerator.cs

```diff
@@ -10,13 +10,13 @@ namespace LoRaTools
 
     public class OTAAKeysGenerator
     {
+        private static readonly Random RndDevAddr = new Random();

```

Comment:
Is static random considered as best practice? Should we think about using the same one across the app or that would generate a similar random value?

Reply:
The problem with using a new instance everytime is that the implementation uses the system timer as a seed. That resolution is not high enough to guarantee unique numbers if you hit that code very quickly, you could end up with the same value. 
In general, we could have a wrapper class that is using a static and is using a lock internally to provide the values.